pages/02.administer/15.admin_guide/55.upgrade/15.buster_bullseye/buster_bullseye_migration.fr.md

```diff
@@ -82,9 +82,9 @@ Pour cela, vous pouvez aller dans la partie Diagnostic (section Système de base
 Vérifiez que vos applications installées fonctionnent... Si elles ne fonctionnent pas, il est recommandé de tenter de les mettre à jour. (ou bien de manière générale, il est recommandé de les mettre à jour même si elles fonctionnent !).
 
 
-Si votre app est cassée et que vous étiez déjà sur la dernière version d'une application, vous pouvez relancer la mise à jour grâce à l'option `-f`:
+Si votre app est cassée et que vous étiez déjà sur la dernière version d'une application, vous pouvez relancer la mise à jour grâce à l'option `-F`:
 ```
-yunohost app upgrade -F NOM_APP
+yunohost app upgrade --force NOM_APP
```

Comment:
It's a bit weird to switch from `-F` in the text to the full flag `--force` in the command.